shellbase-slack/pom.xml

```diff
@@ -31,16 +31,16 @@
 
     <dependency>
       <groupId>com.flyberrycapital</groupId>
-      <artifactId>scala-slack_2.11</artifactId>
-      <version>0.2.0</version>
+      <artifactId>scala-slack_${scala.version.major}</artifactId>
```

Comment:
For the record: bumping this is considered an API change, so we'll need to bump minor version on release.